report_thesis/src/sections/proposed_approach/proposed_approach.tex

```diff
@@ -1,13 +1,15 @@
 \section{Proposed Approach}
-To address the challenges in predicting major oxide compositions from \gls{libs} data, we propose the development of advanced computational models capable of effectively handling the multifaceted challenges arising from the high dimensionality of spectral data, multicollinearity, matrix effects, and limited data availability.
+To address the challenges in predicting major oxide compositions from \gls{libs} data, we propose the development of advanced computational models capable of effectively handling the multifaceted challenges we describe in \ref{subsec:challenges}.
 These issues complicate the accurate and robust prediction of elemental concentrations, necessitating advanced computational methodologies. 
 
 Our approach aims to enhance the prediction accuracy and robustness for major oxides in \gls{libs} data by leveraging specific combinations of machine learning models and preprocessors that are particularly effective at predicting individual oxides.
-The models will use feature vectors $\mathbf{x} \in \mathbb{R}^N$, derived from the Masked Intensity Tensor $\mathbf{M}[\chi, l, \lambda]$ after processing by one or more preprocessors, and output Estimated Concentration Vectors $\mathbf{v} \in \mathbb{R}^{n_o}$. 
-For each model, we calculate a vector of prediction errors $\mathbf{e}$, where each entry $e_i$ represents the difference between the actual and predicted concentration for the $i$-th oxide. Subsequently, for each oxide, we aggregate the corresponding prediction errors across all models to construct a vector. 
-This results in $n$ vectors, each containing prediction errors for a specific oxide from all models.
-Finally, we fit a mapping function $\mathcal{F}$ to the prediction error vectors for each oxide, to produce a final prediction for that oxide.
-We use metrics such as \gls{rmse} for accuracy and the standard deviation of prediction errors for robustness to evaluate our models.
+The models will use feature vectors $\mathbf{x} \in \mathbb{R}^N$ where $N$ is the number of features, derived from the Masked Intensity Tensor $\mathbf{M}[\chi, l, \lambda]$ after processing by one or more preprocessors, and output Estimated Concentration Vectors $\mathbf{v} \in \mathbb{R}^{n_o}$. 
```

Comment:
Sentence gets a little long and hard to read. I also removed ` after being processed by one or more preprocessors.`

```suggestion
The models will use feature vectors $\mathbf{x} \in \mathbb{R}^N$ derived from the Masked Intensity Tensor $\mathbf{M}[\chi, l, \lambda]$ as input, where $N$ is the number of features. 
The output will be Estimated Concentration Vectors $\mathbf{v} \in \mathbb{R}^{n_o}$.
```